src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+
+.calendar {
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $paddings;
+  max-width: $calendar-width;
+  perspective: $cell-size;
+
+  &__day {
+    box-sizing: border-box;
+    @include cell($cell-size);
+    border: 1px solid black;
+    background-color: #eee;
+    text-align: center;
+    line-height: $cell-size;
+    transition: $transition-time;
```

Comment:
To leave "transition-property" blank is the same as to pass there "all", which is not OK. It's better to list them like "background $transition-time, transform $transition-time".

Reply:
Fix it

Reply:
fixed